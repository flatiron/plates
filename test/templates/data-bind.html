<div data-bind="name"></div>
