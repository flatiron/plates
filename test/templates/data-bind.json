{ "name" : "Frank", "placeholder" : "Some dude" }