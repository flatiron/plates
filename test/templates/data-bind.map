{"name" : ["data-bind", "value"] ,"placeholder" : ["data-bind-placeholder", "placeholder"] }