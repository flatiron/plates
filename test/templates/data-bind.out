<div data-bind="name">frank</div>
